app/users/__init__.py

```diff
@@ -41,3 +42,19 @@ def post(self):
             return utils.make_validation_error(e)
         
         return utils.make_response(data=user.to_dict())
+
+    @authenticated_request
+    def get(self, id = None, user = None):
+        """
+        Returns the requested user
+        """
+
+        user = user_service.find_by_id(id)
+
+        if not user:
+            message = 'User cannot be found'
```

Comment:
Let's be a bit more specific and say 'User with ID [requested_id] cannot be found'. You'd do something like `message = 'User with ID %s cannot be found'`